.github/workflows/live-test.yml

```diff
@@ -1,24 +1,70 @@
 name: Live tests for B2K
 on:
-  push:
-    branches: [ "main" ]
-  pull_request:
-    branches: [ "main" ]
+  repository_dispatch:
+    types: [ok-to-test-command]
+permissions:
+  id-token: write
 jobs:
   liveTest:
+    env:
+      PR_NUMBER: ${{ github.event.client_payload.slash_command.args.named.PR_NUMBER }}
```

Comment:
@cxznmhdcxz This env is from /ok-to-test command so that user doesn't have to specify the sha argument and it always get's the latest sha from the branch and runs it correct ? Just adding here as FYI for others to know.

Reply:
I left a comment on this as well. Without the sha this opens us up to a security vulnerability. Good callout!

Reply:
this is updated to input the commit sha.